projects/valtimo/dossier/src/lib/components/dossier-detail/tab/progress/progress.component.css

```diff
@@ -18,3 +18,14 @@
   height: calc(100vh - 380px);
   width: 100%;
 }
+
+.process-info-row {
+  display: flex;
+  flex-direction: row;
```

Comment:
flex-direction is row by default, no need to specify it

Reply:
Changed